.prettierrc.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
```

Comment:
Why change from the existing `.prettierrc` file?

Reply:
I've  seen you using the `.js`  file in other projects so figured I'd use it here too.